### PR TITLE
Bug/EUI 8467

### PR DESCRIPTION
**Pull request by DavidJayakumar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-8467

### Change description ###

Fix the combinations of event and summary

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by MateiHMCTS — CHANGES_REQUESTED** [review]
I don't understand why there is a need for slice and then two reverses? Wouldn't the two reverses cancel out?

**Comment by DavidJayakumar** [comment]
> I don't understand why there is a need for slice and then two reverses? Wouldn't the two reverses will cancel out?

I took this logic from from [here](https://stackoverflow.com/questions/2218999/how-to-remove-all-duplicates-from-an-array-of-objects) If we are not using this logic, it retains the last object, so the order of the array gets changed and hence tabs are not apppering properly.

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@DavidJayakumar - changes look good; would be good if there's a unit test for the `loadCaseDetails()` function in `case-viewer.component.ts`

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit/case-edit.component.ts [hunk 1]
```diff
@@ -409,7 +409,7 @@ private caseSubmit({ form, caseEventData, submit }: CaseEditCaseSubmit ): void {
         this.sessionStorageService.removeItem('eventUrl');
         const confirmation: Confirmation = this.buildConfirmation(response);
         if (confirmation && (confirmation.getHeader() || confirmation.getBody())) {
-          this.confirm(confirmation);
+          this.confirm(confirmation).then().catch();
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit/case-edit.component.ts:412]: @DavidJayakumar - empty `catch()` clause? If intentional, please consider adding a comment to explain

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@DavidJayakumar - one other point to look at

**Review by MateiHMCTS — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**